mongodb/Acornfile

```diff
@@ -0,0 +1,585 @@
+args: {
+    // Specify whether MongoDB instance is a standalone or replica set
+    isReplicaSet: false
+    // Custom user to be created during the initialization
+    dbUserName: ""
+    // Custom database to be created during the initialization
+    dbName: "acorn"
+    // Number of secondary nodes
+    replicas: 3
+    // Enable diagnostic mode
+    diagnosticMode: false
+    // Enable authentication
+    authEnabled: true
+    // Mongodb additional command line flags
+    extraFlags: ""
+    // Run arbiter node
+    arbiterEnabled: false
+    // Arbiter additional command line flags
+    arbiterExtraFlags: ""
+    // Number of hidden nodes
+    hiddenReplicas: 0
+    // Hidden node additional command line flags
+    hiddenExtraFlargs: ""
+    // App installed namespace. NOTE(joseb): before fieldRef is supported,
+    appNamespace: "mongo"
+    // Backup Schedule
+    backupSchedule: ""
+    // Specify to use oplog for taking a point-in-time snapshot
+    ptrBackup: false
+    // Collection name to backup
+    backupCollection: ""
+    // DB name to backup
+    backupDB: ""
+    // Backup name to restore
+    backupToRestore: ""
+    // Mongodb version
+    dbVersion: "5.0.12"
+    // Days to keep backup
+    backupRetainDays: 5
+    // Allow access from outside of k8s cluster
+    externalAccessEnabled: false
+    // TLS certificate
+    tlsCertificate: ""
+    // TLS key
+    tlsKey: ""
+    // TLS CA
+    tlsCA: ""
+    // TLS mode (options: `allowTLS`, `preferTLS`, `requireTLS`)
+    tlsMode: "requireTLS"
+}
+
+profiles: {
+    prod: {
+        backupSchedule: "*/30 * * * *"
+        isReplicaSet: true
+    }
+}
+
+localData: {
+    autoDiscovery: false
+    tlsEnabled: args.tlsCA != "" && args.tlsCertificate != "" && args.tlsKey != ""

```

Comment:
Maybe key this off of tlsMode "disabled" or "off"?

Reply:
Available values for tlsmode are `allowTLS`, `preferTLS`, and `requireTLS`, not on/off.

Reply:
OK, the issue I'm trying to address is none of the args for TLS should be there. If the user isn't doing TLS then they need a way to turn it off. The tlsMode seems like a decent value to key off of. Maybe empty string or something.

Reply:
yeap, like `tlsEnabled: args.tlsMode != ""`